static/src/javascripts/projects/commercial/modules/mobile-sticky.js

```diff
@@ -0,0 +1,45 @@
+// @flow
```

Comment:
Do we have any tests for the mobile sticky? Even a couple would be helpful to document the intended behaviour

Reply:
We don't at the moment but will add them at a later date (team priorities). It will not be forgotten 🙂